src/components/SlateEditor/plugins/embed/SlateVideo.jsx

```diff
@@ -35,50 +59,34 @@ class SlateVideo extends React.PureComponent {
       onRemoveClick,
       ...rest
     } = this.props;
-    const src = `//players.brightcove.net/${config.brightCoveAccountId}/${
-      config.brightcovePlayerId
-    }_default/index.min.js`;
+    const { iframeData } = this.state;
+
     return (
-      <Figure id={embed.videoid} {...attributes}>
+      <Figure id={embed.videoid} resizeIframe {...attributes}>
```

Comment:
Fungerer egentlig resizeIframe? For nå endrer bare bredden seg når man minsker skjermen, i ndla-frontend er er ratio fastlåst.